internal/operator-controller/rukpak/render/certproviders/certmanager.go

```diff
@@ -165,6 +166,9 @@ func (p CertManagerCertificateProvider) AdditionalObjects(cfg render.Certificate
 			Duration: &metav1.Duration{
 				Duration: olmv0RotationPeriod,
 			},
+			RenewBefore: &metav1.Duration{
+				Duration: 24 * time.Hour,
```

Comment:
Could we have a const with the value like we have `olmv0RotationPeriod`?

Reply:
Fixed it ^^